python_on_whales/components/buildx/cli_wrapper.py

```diff
@@ -465,6 +468,8 @@ def create(
 
         full_cmd.add_simple_arg("--buildkitd-flags", buildkitd_flags)
         full_cmd.add_simple_arg("--config", config)
+        if platforms is not None:
+            full_cmd += ["--platform", ",".join(platforms)]
```

Comment:
Can you add a small unit test with multiple platforms to see if it works? Thanks :)